### PR TITLE
Component policies in environment

### PR DESCRIPTION
**Pull request by chemikadze**

### CODE REVIEW

File: qubell/api/private/environment.py [hunk 1]
```diff
@@ -263,12 +280,14 @@ def remove_marker(marker):
 
         actions = dict(clean=clean, add_policy=set_policy, remove_policy=remove_policy, add_marker=add_marker,
                        remove_marker=remove_marker, add_property=set_property, remove_property=remove_property,
-                       add_service=add_service, remove_service=remove_service)
+                       add_service=add_service, remove_service=remove_service,
+                       add_component_policy=set_component_policy, remove_component_policy=remove_component_policy)
```
Comment (dmakhno) [inline qubell/api/private/environment.py]: add_policy/property is called from historical reasons. 
let's leave set_component_policy for outers.

File: qubell/api/private/environment.py [hunk 2]
```diff
@@ -208,6 +210,21 @@ def remove_policy(name):
             data['policies'].remove(policy[0])
             log.info("Removing policy %s from environment %s (%s)" % (name, env_name, self.id))
 
+        def set_component_policy(matchers=list(), actions=list()):
+            print matchers, actions
```
Comment (dmakhno) [inline qubell/api/private/environment.py]: we don't use print at all, please remove

File: qubell/api/private/environment.py [hunk 3]
```diff
@@ -320,6 +339,12 @@ def add_policy(self, policy=None, action=None, parameter=None, value=None):
     def remove_policy(self, policy_name):
         pass
 
+    def add_component_policy(self, matchers, actions):
```
Comment (dmakhno) [inline qubell/api/private/environment.py]: please add these for env,  as add/remove_marker on LL144-150

File: qubell/api/private/environment.py (hunk 2 above)
Comment (chemikadze) [inline qubell/api/private/environment.py]: oops

File: qubell/api/private/environment.py [hunk 4]
```diff
@@ -263,12 +280,14 @@ def remove_marker(marker):
...
                        remove_marker=remove_marker, add_property=set_property, remove_property=remove_property,
-                       add_service=add_service, remove_service=remove_service)
+                       add_service=add_service, remove_service=remove_service,
+                       add_component_policy=set_component_policy, remove_component_policy=remove_component_policy)
 
 
         for operation in env_operations:
             action, args, kwargs = operation
             actions[action](*args, **kwargs)
+        print data
```
Comment (vasichkin) [inline qubell/api/private/environment.py]: print

**Comment by chemikadze** [comment]
@dmakhno @vasichkin remarks fixed

**Pull request merged**